experimental/juvix/contract.jvx

```diff
@@ -0,0 +1,198 @@
+Map = Data.Map
```

Comment:
We are assuming inference, here, yes?

(so there could also be a signature `Map : Type`)

Reply:
Well Map here, isn't of type Type, it's of type Module!

Reply:
Ah right - still, should there be a signature?